packages/Sylius/src/Rector/Review/ReplaceCreateMethodWithoutReviewerRector.php

```diff
@@ -60,15 +60,13 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
-        if ($this->methodArgumentAnalyzer->hasMethodNthArgument($node, 2)
-            && ! $this->methodArgumentAnalyzer->isMethodNthArgumentNull($node, 2)
-        ) {
+        if (isset($node->args[1]) && ! $this->isNull($node->args[1])) {
             return null;
         }
 
         $this->identifierRenamer->renameNode($node, 'createForSubject');
 
-        if ($this->methodArgumentAnalyzer->hasMethodNthArgument($node, 2)) {
+        if (isset($node->args[1])) {
```

Comment:
To keep it consistent with `$node->params`, `$node->stmts`, `$node->args` where keys are form 0. The start with 1 is confusing.